tests/test_clos/test_detect_workflow.py

```diff
@@ -0,0 +1,40 @@
+"""Pytest for fucntion detect_workflow"""
```

Comment:
```suggestion
"""Pytest for method Cloudos.detect_workflow"""
```
Although it could be removed for this simple test file.